tests/Stub/SessionStub.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
This class is identical to `Session` itself, why re-creating it instead of reusing `Session`?

Reply:
@bryanjhv  Morning

Unit testing happend on php-cli, I do not think CLI mode would have session so I remove ```session_start()``` and I overwrite ```startSession()``` method member access to public, then I can call ```startSession()``` directly but not something like:
``` php
$session = new Session();
$session();
//or
$session->__invoke();
//but
$session->startSession();
```
After all, I am do unit test for startSession method, that's why I need a stub for ```Session.php```

Reply:
I think for now we can ignore tests, because that would involve integration with a Slim app.

By the way, PHP CLI has `session_start()` defined and works at least for the CLI session.

Reply:
```php
php > session_start();
PHP Warning:  session_start(): Cannot send session cookie - headers already sent by (output started at php shell code:1) in php shell code on line 1
PHP Stack trace:
PHP   1. {main}() php shell code:0
PHP   2. session_start() php shell code:1

Warning: session_start(): Cannot send session cookie - headers already sent by (output started at php shell code:1) in php shell code on line 1

Call Stack:
 6337.2755     351304   1. {main}() php shell code:0
 6337.2755     351304   2. session_start() php shell code:1

PHP Warning:  session_start(): Cannot send session cache limiter - headers already sent (output started at php shell code:1) in php shell code on line 1
PHP Stack trace:
PHP   1. {main}() php shell code:0
PHP   2. session_start() php shell code:1

Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at php shell code:1) in php shell code on line 1

Call Stack:
 6337.2755     351304   1. {main}() php shell code:0
 6337.2755     351304   2. session_start() php shell code:1
```